pyuvdata/uvbase.py

```diff
@@ -275,6 +297,17 @@ def __eq__(self, other, check_extra=True):
                         f" right is {other_param.value}."
                     )
                     p_equal = False
+
+            if allowed_failures is not None:
+                for p in allowed_failures:
+                    self_param = getattr(self, p)
+                    other_param = getattr(other, p)
+                    if self_param != other_param:
+                        print(
+                            f"parameter {p} does not match. Left is {self_param.value},"
+                            f" right is {other_param.value}."
+                        )
```

Comment:
should we add something to this to clarify that this is an allowed failure so people don't worry about it?

Reply:
I think it could be helpful, otherwise it just looks like the failure of required parameters

Reply:
Okay, I can tweak the message that's printed to make it explicit that it's okay they're different.

Reply:
Updated in 00cb301.